_posts/en/posts/2015-12-23-capacity-increases-faq.md

```diff
@@ -16,8 +16,9 @@ New technology will be deployed when it is ready and has been tested. However, w
 | Feb 2016 | 0.12.0 | [libsecp256k1 verification][] |
 | Feb 2016 | | Segregated witness feature complete & ready for general review |
 | Mar 2016\* | 0.12.x | Deploy OP_CHECKSEQUENCEVERIFY (BIPs [68][BIP68] & [112][BIP112]) + [BIP113][] as first [BIP9][] versionbits soft fork |
-| April 2016\* |  0.12.x |  Deploy segregated witness |
-| 2016 | | Weak blocks, IBLTs, or both |
+| April 2016\* |  0.12.x |  Deploy segregated witness (including block size increase) |
+| 2016 | | Weak blocks and IBLT, Lightning, or both |
+| 2017 | | Prepare for hardfork |
```

Comment:
This is technically out of date if we're going to be looking at potential HF proposals by July 2016